FloripaBus/Repositories/IRouteRepository.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using Newtonsoft.Json;
+using System.Net;
+
+namespace FloripaBus

```

Comment:
I noticed that in this file you added a bunch of classes. Why didn't you separate each class in one different file? Can you explain that?